custom_components/tahoma/select.py

```diff
@@ -39,6 +39,16 @@
         ),
         entity_category=EntityCategory.CONFIG,
     ),
+    OverkizSelectDescription(
+        key=OverkizState.OVP_HEATING_TEMPERATURE_INTERFACE_OPERATING_MODE,
+        name="Operating mode",
+        icon="mdi:sun-snowflake",
+        options=[OverkizCommandParam.HEATING, OverkizCommandParam.COOLING],
+        select_option=lambda option, execute_command: execute_command(
+            "setOperatingMode", option
```

Comment:
Did you add setOperatingMode to pyoverkiz already?

Reply:
Yes, will push the change.